packages/swapper/src/swappers/thorchain/utils/getPriceRatio/getPriceRatio.ts

```diff
@@ -52,6 +52,17 @@ export const getPriceRatio = async (
       return bn(runeBalance).dividedBy(sellAssetBalance).toString()
     }
 
+    if (isRune(sellAssetId) && buyPool) {
```

Comment:
This whole file is an absolute spaghetti-filled dumpster fire with innumerable conditional returns and rune-specific logic sprinkled throughout - and this PR only adds to it.